File: code-coaches/medium/tax-free/TaxFree.java
import java.util.Scanner;

public class TaxFree {

    public static float getTotal(int[] prices) {
        float tax = 1.07f;
        float total = 0f;
        for (int price : prices) {
            total += (price < 20) ? price * tax : price;
        }
        float totalRounded = (float) Math.round(total * 100) / 100;
        return totalRounded;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String input = scanner.nextLine();
        String[] priceStrings = input.split(",");

        int[] prices = new int[priceStrings.length];
        for (int i = 0; i < priceStrings.length; i++) {
            prices[i] = Integer.parseInt(priceStrings[i]);
        }

        System.out.printf("%.2f%n", getTotal(prices));

        scanner.close();
    }

}

File: code-coaches/medium/tax-free/TaxFree.cs
using System;

public class TaxFree
{
    static float GetTotal(int[] prices)
    {
        float tax = 1.07f;
        float total = 0f;
        foreach (int price in prices)
        {
            total += (price < 20) ? price * tax : price;
        }
        float totalRounded = (float)Math.Round(total * 100) / 100;
        return totalRounded;
    }

    static void Main(string[] args)
    {
        string input = Console.ReadLine();
        string[] priceStrings = input.Split(',');

        int[] prices = new int[priceStrings.Length];
        for (int i = 0; i < priceStrings.Length; i++)
        {
            prices[i] = int.Parse(priceStrings[i]);
        }

        Console.WriteLine($"{GetTotal(prices):F2}");
    }
}
